packages/core/src/utils/react-query-utils.ts

```diff
@@ -146,7 +147,27 @@ export const retryFunction = (failureCount: number, error: any) => {
   return false
 }
 
-export const defaultQueryConfig = {
-  suspense: true,
-  retry: retryFunction,
+export function getDefaultQueryConfig() {
+  // Default blitz reactMode
+  let reactMode = "concurrent"
+
+  // On the server side load reactMode from the config
+  if (isServer) {
+    const config = getConfig()
+
+    if (config.experimental?.reactMode) {
+      reactMode = config.experimental.reactMode
+    }
+  } else {
+    // On the client side load reactMode from the env variable
+    if (process.env.NEXT_PUBLIC_REACT_MODE) {
+      reactMode = process.env.NEXT_PUBLIC_REACT_MODE
+    }
+  }
+
+  return {
+    // Set default value for suspense option to true only when concurrent react mode is active
+    suspense: reactMode === "concurrent" ? true : undefined,
```

Comment:
Is this correct? I think Suspense is also supported in "blocking" mode.  This check might be better written as `reactMode !== "legacy"`.